.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Gitter Chat
+    url: https://gitter.im/VowpalWabbit/community/
+    about: Please ask questions here.
```

Comment:
This should be `An IM communication channel for discussing doubts, new features and everything related to VowpalWabbit`